runtime/lua/vscode-neovim/highlight.lua

```diff
@@ -55,7 +55,11 @@ local function setup_globals()
     Visual       = {},
     VisualNC     = {},
     VisualNOS    = {},
-    Substitute   = {},
+    -- By default, it's linked to Search.
+    -- But Search highlighting defaults to using VSCode ThemeColor, potentially with some opacity.
+    -- When the decoration is "virtText", having a background color with opacity can
+    -- cause the virtual text to blend with the original content.
+    Substitute   = { fg='#282c34', bg='#98c379' },
```

Comment:
Nice! We should offer a way to customize these colors if possible, though. I can imagine this color not showing very well on some light themes.

Reply:
Now it's possible, the `setup_globals` will only execute once, and users can define it themselves, requiring some delay for custom highlighting. However, indeed, there is a need to provide a more conventional approach.

Reply:
Gotcha, I guess I was thinking of allowing it via our configuration, but neovim globals work too

Reply:
https://github.com/microsoft/vscode/issues/32813